app/controllers/carts_controller.rb

```diff
@@ -15,4 +17,8 @@ def add
   def find_cart
     @cart = Cart.find(session[:cart_id])
   end
+
+  def show
+    @shopping_cart_items = find_cart.shopping_cart_items
```

Comment:
Does this actually return a collection?

Reply:
If I do a binding.pry then this returns me the items in the cart, just kind of stuck how to show it on the site